.circleci/config.yml

```
@@ -390,6 +391,59 @@ jobs:
           key: cargocache-v2-contract_ibc_reflect-rust:1.47.0-{{ checksum "Cargo.lock" }}
 
 
+  contract_ibc_reflect_send:
+    docker:
+      - image: rust:1.47.0
+    environment:
+      RUST_BACKTRACE: 1
+    working_directory: ~/cosmwasm/contracts/ibc-reflect-send
+    steps:
+      - checkout:
+          path: ~/cosmwasm
+      - run:
+          name: Version information
+          command: rustc --version; cargo --version; rustup --version
+      - restore_cache:
+          keys:
+            - cargocache-v2-contract_ibc_reflect-send-rust:1.47.0-{{ checksum "Cargo.lock" }}
```

This always includes the job name

```suggestion
            - cargocache-v2-contract_ibc_reflect_send-rust:1.47.0-{{ checksum "Cargo.lock" }}
```